lib/backends/sdc/containers.js

```diff
@@ -1846,6 +1749,66 @@ function buildVmPayload(opts, container, callback) {
             cb();
         },
 
+        function handleSharedVolumes(_, cb) {
+            if (opts.config.experimental_nfs_shared_volumes !== true) {
+                cb();
+                return;
+            }
+
+            log.debug('handling shared volumes');
+
+            binds = container.Binds
+                || container.HostConfig && container.HostConfig.Binds;
+
+            if (binds && binds.length > 0) {
+                var volumeComponents = binds[0].split(':');
```

Comment:
> A comment here explaining what this "binds" or "binds[0]" are actually expected to look like might be helpful.

Absolutely, I will add a comment.

> We're also ignoring options for now too, right? E.g. -v myvolume:/mymnt:ro?

For now yes, this is part of the follow-up tickets/change that I'll file when/if that PR gets merged.